internal/app/app.go

```diff
@@ -229,7 +235,7 @@ func (app *App) externalCAFileChecker(ctx context.Context) {
 		select {
 		case <-ticker.C:
 			localNode := app.cluster.Local()
-			replicaStatus, err := localNode.GetExternalReplicaStatus()
+			replicaStatus, err := app.externalReplication.GetReplicaStatus(localNode)
```

Comment:
If we have External Replication turned off - we should not run externalCAFileChecker goroutine